src/orion/algo/hyperband.py

```diff
@@ -319,18 +373,18 @@ def suggest(self, num):
 
         Returns
         -------
-        list of points or None
-            A list of lists representing points suggested by the algorithm. The algorithm may opt
-            out if it cannot make a good suggestion at the moment (it may be waiting for other
-            trials to complete), in which case it will return None.
+        list of `orion.core.worker.trial:Trial` or None
+            A list of trials suggested by the algorithm. The algorithm may opt out if it cannot make
+            a good suggestion at the moment (it may be waiting for other trials to complete), in
+            which case it will return None.
 
         """
         self._refresh_brackets()
 
         samples = self.promote(num)
 
         samples.extend(self.sample(max(num - len(samples), 0)))
-
+        assert self.brackets is not None
```

Comment:
Another

Reply:
Fixed in 35ce6867769d77028c87f7bb015adfd349315f85